seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpGetCommandProcessor.java

```diff
@@ -692,6 +693,13 @@ private JsonObject convertToJson(JobInfo jobInfo, long jobId) {
             jobStatus = seaTunnelServer.getCoordinatorService().getJobStatus(jobId);
         }
 
+        JobDAGInfo jobDAGInfo =
```

Comment:
can we use https://github.com/apache/seatunnel/blob/511c8afad4e7b4a4e29db532136878305e44b1e1/seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/master/JobMaster.java#L436 ?

Reply:
> can we use
> 
> https://github.com/apache/seatunnel/blob/511c8afad4e7b4a4e29db532136878305e44b1e1/seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/master/JobMaster.java#L436
> 
> ?

Can't get JobMaster here.

Reply:
ok